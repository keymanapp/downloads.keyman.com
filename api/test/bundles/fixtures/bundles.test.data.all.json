{
    "acoli": {
        "1.0": [
            "acoli.keyboard_info",
            "acoli.kmp",
            "keymandesktop-10.0.1207.0-acoli-1.0.exe",
            "keymandesktop-11.0.1361.0-acoli-1.0.exe",
            "keymandesktop-12.0.52.0-acoli-1.0.exe"
        ]
    },
    "africadeadkey": {
        "1.0": [
            "africadeadkey.keyboard_info",
            "africadeadkey.kmp",
            "keymandesktop-10.0.1207.0-africadeadkey-1.0.exe",
            "keymandesktop-11.0.1361.0-africadeadkey-1.0.exe",
            "keymandesktop-12.0.52.0-africadeadkey-1.0.exe"
        ]
    },
    "basic_kbdinen": {
        "1.0": [
            "basic_kbdinen.js",
            "basic_kbdinen.keyboard_info",
            "basic_kbdinen.kmp",
            "keymandesktop-10.0.1207.0-basic_kbdinen-1.0.exe",
            "keymandesktop-11.0.1361.0-basic_kbdinen-1.0.exe",
            "keymandesktop-12.0.52.0-basic_kbdinen-1.0.exe"
        ]
    }
}