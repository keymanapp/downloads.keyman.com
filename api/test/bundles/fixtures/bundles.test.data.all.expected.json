{
    "acoli": {
        "1.0": {
            "windows": {
                "10.0.1207.0": {
                    "url": "/keyboards/acoli/1.0/keymandesktop-10.0.1207.0-acoli-1.0.exe"
                },
                "11.0.1361.0": {
                    "url": "/keyboards/acoli/1.0/keymandesktop-11.0.1361.0-acoli-1.0.exe"
                },
                "12.0.52.0": {
                    "url": "/keyboards/acoli/1.0/keymandesktop-12.0.52.0-acoli-1.0.exe"
                }
            }
        }
    },
    "africadeadkey": {
        "1.0": {
            "windows": {
                "10.0.1207.0": {
                    "url": "/keyboards/africadeadkey/1.0/keymandesktop-10.0.1207.0-africadeadkey-1.0.exe"
                },
                "11.0.1361.0": {
                    "url": "/keyboards/africadeadkey/1.0/keymandesktop-11.0.1361.0-africadeadkey-1.0.exe"
                },
                "12.0.52.0": {
                    "url": "/keyboards/africadeadkey/1.0/keymandesktop-12.0.52.0-africadeadkey-1.0.exe"
                }
            }
        }
    },
    "basic_kbdinen": {
        "1.0": {
            "windows": {
                "10.0.1207.0": {
                    "url": "/keyboards/basic_kbdinen/1.0/keymandesktop-10.0.1207.0-basic_kbdinen-1.0.exe"
                },
                "11.0.1361.0": {
                    "url": "/keyboards/basic_kbdinen/1.0/keymandesktop-11.0.1361.0-basic_kbdinen-1.0.exe"
                },
                "12.0.52.0": {
                    "url": "/keyboards/basic_kbdinen/1.0/keymandesktop-12.0.52.0-basic_kbdinen-1.0.exe"
                }
            }
        }
    }
}